scripts/start_online_demo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+function seedDemoData(){
+    echo 'Seeding data for online demo'
+    bytebase --host http://localhost --port 80 --demo --data /var/opt/bytebase &
```

Comment:
host should be https://demo.bytebase.com 

See https://docs.bytebase.com/reference/command-line#host-less-than-less-than-string-greater-than-greater-than